libs/src/services/creatorNode/CreatorNodeSelection.js

```diff
@@ -1,4 +1,5 @@
 const ServiceSelection = require('../../service-selection/ServiceSelection')
+const CreatorNode = require('../../services/creatorNode/index')
 const { timeRequestsAndSortByVersion } = require('../../utils/network')
```

Comment:
this file will disappear from the diff when https://github.com/AudiusProject/audius-protocol/pull/1218 is merged